### ISSUE TITLE
Individually Exposed USB

### ISSUE THREAD

**Issue by the1maximus** [derek-miller/homebridge-unifi-smartpower#6]
**Is your feature request related to a problem? Please describe:**

No Issues

**Describe the solution you'd like:** Individually exposed USB Power switches. 

 Currently the USB ports are exposed as 1 power switch, I would like to see them exposed individually. 

There are no alternatives currently. 

**Additional context:**

 In the current official firmware release of the UI Console, it shows as 1 switch for the USBs. I’m currently running the latest EA firmware and the USBs are all individually exposed in the console now. I would like to see the USBs exposed individually in HK as well. 

Additionally, I would to say thank you for the great work. This plug-in was something I was hoping I would find when I purchased the USP-Strip, and so far it’s working great. 

I hope you are able to get the plug-in verified as it is a great addition for anyone running UI hardware.

**Comment by the1maximus** [comment]
I did, I have to digress. It looks like the update that exposed the 4 USB was pulled back, as it’s only showing 1  switch for the USB now. 

Hopefully when it does get the official update it’s corrected and they’ll be exposed. Thanks again for the work you put into this.

**Comment by the1maximus** [comment]
@derek-miller Figured I’d make mention that I noticed I was having an issue where even though the plugs were exposed, It wouldn’t let me control them. Turns out it was the permissions I had set up. 

Instead of using my admin privileges, I made a local only account that had view access only (I have my protect plug-in setup the same way). Well turns out view only does not allow for the plugs to be controlled. I switched it to a “Site Admin” role and now it works perfectly.
